ITGlueAPI/ITGlueAPI.psd1

```diff
@@ -107,13 +107,15 @@
     FunctionsToExport = 'Add-ITGlueAPIKey',
                         'Get-ITGlueAPIKey',
                         'Remove-ITGlueAPIKey',
+                        'Test-ITGlueAPIKey',
```

Comment:
Once #157 is complete and if this is something that has value to add to the module I can migrate the `Invoke` section to the new internal API call function(s).